test/unit/lib/censorPartnership.js

```diff
@@ -0,0 +1,304 @@
+'use strict';
+const { censorPartnership } = require('../../../src/lib/censorRecords');
+const roleWithFullAccess = ['full-details'];
+const roleWithoutFullAccess = ['births'];
+
+const partnershipData = {
```

Comment:
Now that I think of it...

I wonder if we should make use of https://github.com/UKHomeOffice/lev-api/blob/master/mock/partnership_registration_v1.json

Forgot we had that! :-/

Reply:
The data is in a different 'shape' and doesn't have any blocked records - I think the work required to refactor would be more than it's worth.